app/src/main/java/org/gdg/frisbee/android/utils/PrefUtils.java

```diff
@@ -3,37 +3,150 @@
 import android.content.Context;
 import android.content.SharedPreferences;
 
+import org.joda.time.DateTime;
+
 public final class PrefUtils {
-    private static final String PREF_NAME = "gdg";
-    private static final String PREF_OPEN_DRAWER_ON_START = "open_drawer_on_start";
-    private static final String PREF_SIGNED_IN = "gdg_signed_in";
+    public static final String PREF_NAME = "gdg";
+    public static final String SETTINGS_HOME_GDG = "gdg_home";
+    public static final String SETTINGS_GCM = "gcm";
+    public static final String SETTINGS_SIGNED_IN = "gdg_signed_in";
+    public static final String SETTINGS_ANALYTICS = "analytics";
+    private static final String PREFS_VIDEOS_PLAYED = "gdg_app_videos_played";
+    private static final String PREFS_OPEN_DRAWER_ON_START = "open_drawer_on_start";
+    private static final String PREFS_FIRST_START = "gdg_first_start";
+    private static final String PREFS_GCM_REG_ID = "gdg_registration_id";
+    private static final String PREFS_VERSION_CODE = "gdg_version_code";
+    private static final String PREFS_APP_STARTS = "gdg_app_starts";
+    private static final String PREFS_GCM_NOTIFICATION_KEY = "gcm_notification_key";
+    private static final String PREFS_SEASONS_GREETINGS = "seasons_greetings";
+
+    private static final boolean PREFS_FIRST_START_DEFAULT = true;
 
     private PrefUtils() {
         // Prevent instances of this class being created.
     }
 
+    public static SharedPreferences prefs(final Context context) {
+        return context.getSharedPreferences(PREF_NAME, Context.MODE_PRIVATE);
```

Comment:
Maybe we can create an object that wraps this SharedPreferences and gives a nice API to access the preferences. In PrefUtils there are too many static methods.